### DISCUSSION TITLE
How disable for blade.php

### DISCUSSION THREAD

**Discussion by olegopro** [izhangzhihao/intellij-rainbow-brackets#2748] [Q&A]
**blade.php** don't work in -> Do NOT rainbowify these languages (name or file extension, comma separated)

**Comment by izhangzhihao** [comment]
try to add 'php' if you didn't want to highlight php files.

**Reply by olegopro** [reply]
I want to disable only blade.php. Is there a way to do this?

**Comment by izhangzhihao** [comment]
Not possible now
________________________________
